pkg/acsengine/defaults.go

```diff
@@ -764,6 +764,9 @@ func setStorageDefaults(a *api.Properties) {
 		if len(profile.AvailabilityProfile) == 0 {
 			profile.AvailabilityProfile = api.VirtualMachineScaleSets
 		}
+		if len(profile.ScaleSetEvictionPolicy) == 0 && profile.ScaleSetPriority == api.ScaleSetPriorityLow {
```

Comment:
Extra credit: add a unit test in default_test.go to make sure EvictionPolicy / Priority are set coherently 🙂

Reply:
Great point. I've just added TestAgentPoolProfile which check default values remain empty, and that the ScaleSetEvictionPolicy is set correctly when ScaleSetPriority is Low.